dashboard/client/src/pages/overview/cards/RecentServeCard.tsx

```diff
@@ -0,0 +1,174 @@
+import { createStyles, makeStyles } from "@material-ui/core";
+import classNames from "classnames";
+import _ from "lodash";
+import React from "react";
+import {
+  RiCheckboxCircleFill,
+  RiCloseCircleFill,
+  RiLoader4Line,
+} from "react-icons/ri";
+import { Link } from "react-router-dom";
+import { CommonRecentCard } from "../../../components/CommonRecentCard";
+import { UnifiedJob } from "../../../type/job";
+import { ServeApplication, ServeApplicationStatus } from "../../../type/serve";
+import { useServeApplications } from "../../serve/hook/useServeApplications";
+import { LinkWithArrow, OverviewCard } from "./OverviewCard";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    root: {
+      display: "flex",
+      flexDirection: "column",
+      padding: theme.spacing(2, 3),
+    },
+    listContainer: {
+      marginTop: theme.spacing(2),
+      flex: 1,
+      overflow: "hidden",
+    },
+    listItem: {
+      "&:not(:first-child)": {
+        marginTop: theme.spacing(1),
+      },
+    },
+
+    icon: {
+      width: 24,
+      height: 24,
+      marginRight: theme.spacing(1),
+      flex: "0 0 20px",
+    },
+    "@keyframes spinner": {
+      from: {
+        transform: "rotate(0deg)",
+      },
+      to: {
+        transform: "rotate(360deg)",
+      },
+    },
```

Comment:
IN a recent PR, I created a shared component for this animated spinning icon. Please re-use that component instead.

I also created a `JobStatusIcon` for the job status. Let's create a `ServiceStatusIcon` similar to `JobStatusIcon` and reuse the icon in the Serve Appplications table